examples/organization/variables.tf

```diff
@@ -35,6 +41,12 @@ variable "blocked_users" {
   ]
 }
 
+variable "create_all_members_team" {
+  description = "Whether or not to create a team that contains all users of the organization. The teams name can be set through the all_members_team_name"
+  type        = bool
+  default     = true
```

Comment:
this should trigger version increase to `0.2.x` as it is a new default behavior...

Reply:
but this is only an example ?